packages/components/tests/integration/components/hds/tooltip/index-test.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { focus, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | hds/tooltip/index', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it should render the component with a CSS class that matches the component name', async function (assert) {
+    await render(
+      hbs`<Hds::Tooltip @text="More info." id="test-tooltip">info</Hds::Tooltip>`
+    );
+    assert.dom('#test-tooltip').hasClass('hds-tooltip');
+  });
+
+  // Test Content & accessibility features:
+  test('it renders content passed into the tooltip', async function (assert) {
+    // const escapeKey = 27;
+
+    await render(
+      hbs`<Hds::Tooltip @text="More info." id="test-tooltip">info</Hds::Tooltip>`
+    );
+
+    // Test that tooltip does not display by default:
+    assert.dom('.tippy-box').doesNotExist();
+
+    // Focus button to trigger tooltip display:
+    await focus('#test-tooltip');
+    assert.dom('.tippy-box').exists();
+
+    // Trigger escape key to close the tooltip:
+    // TODO: NOT WORKING
+    // await triggerKeyEvent(
+    //   '#test-tooltip',
+    //   'keyup',
+    //   escapeKey
+    // );
+    // // test that the tooltip is now gone:
+    // assert.dom('.tippy-box').doesNotExist();
```

Comment:
So far I can't get this test to work. (Note: I import `triggerKeyEvent` above when trying it.)

Reply:
Found a fix finally. Had to both add a delay and change the eventType from keyup to keydown.